session/list_test.go

```diff
@@ -0,0 +1,128 @@
+package session_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/arborchat/muscadine/session"
+	"github.com/onsi/gomega"
+)
+
+const (
+	username    = "username"
+	sessionName = "sessionname"
+)
+
+// TestCreateUserList ensures that CreateUserList returns a valid UserList
+func TestCreateUserList(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
```

Comment:
@jwhett There are ways around it, like using [`ginkgo`](https://github.com/onsi/ginkgo) as a test framework, but it would be **yet another** dependency. I've used it in the past. It makes test output pretty and provides a different structure for tests. The downside is being heavier, less standardized, and needing to restructure existing tests to play nicely with it. Perhaps we can mix & match test styles with it, but that has its own problems.